### DISCUSSION TITLE
allow removal of ESC functionality

### DISCUSSION THREAD

**Discussion by AaronOldenburg** [cmoskvitin/obsidian-prozen#11] [Ideas]
I the zen mode so far, it's exactly what I'm looking for. Unfortunately, with Vim keybindings, it's not usable, as it exits every time I hit ESC.

Is it possible to allow removal of ESC functionality, and have the main hotkey that we set be for toggling the mode on and off?

**Comment by rustbuckett** [comment]
I'm in exactly the same boat. I love the plugin, but it's unusable with Vim keybindings.

I agree that a mode toggle keybind would solve this issue.
